djangomako/backends.py

```diff
@@ -164,7 +165,37 @@ def render(self, context=None, request=None):
             context['csrf_input'] = csrf_input_lazy(request)
             context['csrf_token'] = csrf_token_lazy(request)
 
-        return self.template.render(**context)
+        try:
+            return self.template.render(**context)
+        except Exception as e:
+            tb = RichTraceback()
+
+            lines = tb.source
+            if not lines:
+                # There's no templates lines then raise
+                raise e
+
+            lines = lines.split('\n')
+            line = tb.lineno
```

Comment:
rename `line` to `error_target`

Reply:
this variable is named `line` in [django template_debug dict](https://docs.djangoproject.com/en/2.0/topics/templates/#contextual-line-information), then I guess `line` is still the better name to track this variable